flink-runtime/src/main/java/org/apache/flink/runtime/security/SecurityUtils.java

```diff
@@ -230,10 +231,21 @@ private void validate() {
 		}
 
 		private static List<String> parseList(String value) {
-			if(value == null) {
+			if(value == null || value.isEmpty()) {
 				return Collections.emptyList();
 			}
-			return Arrays.asList(value.split(","));
```

Comment:
@EronWright your suggestion will be work. But I'd like using regular expression. The JDK DOC also has the same recommend:
 `StringTokenizer is a legacy class that is retained for compatibility reasons although its use is discouraged in new code. It is recommended that anyone seeking this functionality use the split method of String or the java.util.regex package instead.`
Please see: http://docs.oracle.com/javase/7/docs/api/java/util/StringTokenizer.html
What do you think ? @EronWright @tzulitai 
Best,
SunJincheng